crates/ra_hir_ty/src/tests/traits.rs

```diff
@@ -1,7 +1,10 @@
-use super::{infer, type_at, type_at_pos};
-use crate::test_db::TestDB;
 use insta::assert_snapshot;
+
 use ra_db::fixture::WithFixture;
+use test_utils::covers;
+
+use super::{infer, infer_with_mismatches, type_at, type_at_pos};
+use crate::test_db::TestDB;
```

Comment:
Wow! Are you using intellij-rust to develop this? 😉

Reply:
You mean because of the reordering? I just did it by hand, I use Emacs ;)

Reply:
Yep, and it's surprising 😅 you mean Emacs with rust-analyzer right?

Reply:
Yes